x-pack/plugins/endpoint/public/applications/endpoint/view/alerts/details/metadata/general_accordion.tsx

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React, { memo, useMemo } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiAccordion, EuiDescriptionList } from '@elastic/eui';
+import { Immutable, AlertData } from '../../../../../../../common/types';
+import { FormattedDate } from '../../formatted_date';
+
+export const GeneralAccordion = memo(({ alertData }: { alertData: Immutable<AlertData> }) => {
+  const columns = useMemo(() => {
+    return [
+      {
+        title: i18n.translate('xpack.endpoint.application.endpoint.alertDetails.alertType', {
+          defaultMessage: 'Alert Type',
+        }),
+        description: alertData.event.category,
```

Comment:
make a ticket? i18n

Reply:
Added task to do this here: https://github.com/elastic/endpoint-app-team/issues/92